Makefile

```diff
@@ -233,6 +233,28 @@ deploy-test: _verify_minikube generate-keys
 	export ENABLE_AUTH="True" && export DUMMY_IGNITION="True" && \
 	$(MAKE) _update-minikube deploy-wiremock deploy-all
 
+generate-onprem-environment:
+	echo "POSTGRESQL_DATABASE=installer" > onprem-environment
+	echo "POSTGRESQL_PASSWORD=admin" >> onprem-environment
+	echo "POSTGRESQL_USER=admin" >> onprem-environment
+	echo "DB_HOST=127.0.0.1" >> onprem-environment
+	echo "DB_PORT=5432" >> onprem-environment
+	echo "DB_USER=admin" >> onprem-environment
+	echo "DB_PASS=admin" >> onprem-environment
+	echo "DB_NAME=installer" >> onprem-environment
+	echo "SERVICE_BASE_URL=http://127.0.0.1:8090" >> onprem-environment
+	echo "DEPLOY_TARGET=onprem" >> onprem-environment
+	echo "DUMMY_IGNITION=false" >> onprem-environment
+	echo "OPENSHIFT_INSTALL_RELEASE_IMAGE=${OPENSHIFT_INSTALL_RELEASE_IMAGE}" >> onprem-environment
+
+generate-onprem-iso-ignition:
```

Comment:
Can you also update the doc stating there is a new make target to generate the base ignition config file? https://github.com/openshift/assisted-service/blob/master/docs/installer-live-iso.md

Reply:
Done